src/main/java/com/amazon/opendistroforelasticsearch/security/OpenDistroSecurityPlugin.java

```diff
@@ -200,6 +200,7 @@
     private volatile NamedXContentRegistry namedXContentRegistry = null;
     private volatile DlsFlsRequestValve dlsFlsValve = null;
     private volatile Salt salt;
+    public static final String pluginName = "opendistro-security";
```

Comment:
nit: constant use all capital: `PLUGIN_NAME`. Move it to line 182 where other `static` are defined.

Reply:
Updated